kogito-codegen-modules/kogito-codegen-processes-integration-tests/src/test/java/org/kie/kogito/codegen/process/ProcessGenerationIT.java

```diff
@@ -204,7 +204,7 @@ private static void assertNodes(Node[] expected, Node[] current) {
 
     private static final BiConsumer<Node, Node> nodeAsserter = (expected, current) -> {
         assertThat(current.getId()).isEqualTo(expected.getId());
-        if (expected.getName() != null) {
+        if (expected.getName() != null && !expected.getName().trim().isEmpty()) {
```

Comment:
Maybe we could use `StringUtils.isBlank` here like:
```suggestion
        if (!StringUtils.isBlank(expected.getName()) {
) {
```

But I'm ok